src/coreclr/src/md/enc/pdbheap.cpp

```diff
@@ -30,13 +30,13 @@ HRESULT PdbHeap::SetData(PORTABLE_PDB_STREAM* data)
     m_data = new BYTE[m_size];
 
     ULONG offset = 0;
-    IfFailGo(hr = memcpy_s(m_data + offset, m_size, &data->id, sizeof(data->id)));
+    IfFailGo(memcpy_s(m_data + offset, m_size, &data->id, sizeof(data->id)));
```

Comment:
memcpy_s doesn't return `HRESULT`, it returns `errno_t`. So you cannot use IfFailGo for it.

Reply:
Thank you for your remark, this has been fixed with https://github.com/dotnet/runtime/pull/37702/commits/b99bdf5685d18eea23343fe3e638d7f5e5cf41f8